bedrock/base/templates/includes/protocol/navigation/nav-cta.html

```diff
@@ -8,16 +8,19 @@
       {% if not hide_nav_download_button %}
         {{ download_firefox_thanks(alt_copy=ftl('download-button-download-firefox'), dom_id='protocol-nav-download-firefox', button_class='mzp-t-secondary mzp-t-md', download_location='nav') }}
       {% endif %}
-      {% if not hide_nav_get_account_button %}
+      {% if not hide_nav_get_vpn_button %}
       <div class="c-navigation-fxa-cta-container">
-        {{ fxa_button(
-          entrypoint='mozilla.org-globalnav',
-          include_metrics=False,
-          button_text=ftl('navigation-v2-get-a-firefox-account'),
-          class_name='mzp-t-secondary mzp-t-md c-navigation-fxa-cta',
-          optional_parameters={'utm_campaign': 'navigation', 'utm_content': 'get-firefox-account'},
-          optional_attributes={'data-cta-text': 'Get a Firefox Account', 'data-cta-type': 'fxa-sync', 'data-cta-position': 'navigation', 'data-alt-href': url('firefox.accounts')}
-        ) }}
+        {# todo: add tests for fx & non-fx users, analytics attribution #}
+        <a
+          href="{{ url('products.vpn.landing') }}"
```

Comment:
add utm params?

Reply:
not a link to subscribe or sign-in, so should be fine with just the landing page url